.gitignore

```diff
@@ -23,39 +22,11 @@ var/
 .installed.cfg
 *.egg
 
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.coverage
-.coverage.*
-.cache
-nosetests.xml
-coverage.xml
-*,cover
-
-# Translations
-*.mo
-*.pot
-
-# Django stuff:
-*.log
-
 # Sphinx documentation
 docs/_build/
 
-# PyBuilder
-target/
-
-# IDE
-.idea
-.vscode
+# Environments
+env
+venv
+.env
+.venv
```

Comment:
What is the reason behind removing ignore files for IDE(s), coverage, packaging and other temporary files ? It doesn't hurt to have them here unless they are hindering the python wrapper usage is some other application - it helps with users of different environments doing different tasks.

Reply:
Most parts of the ignored files were added by the auto-generation of the `.gitignore` file. Coverage reports are not produced, the Django framework is not used, and so forth. If you add support of, e.g, IDEs, then you should add the support of all IDEs. What about pycharm users? Why only sublime and vscode IDEs are ignored? It's better to keep only the project ignore files in global git ignore and keep IDEs in the local version.

Reply:
It wasn't an explicit choice to include some IDE users or exclude others. Most likely the gitignore file was from github samples added in one of the initial commits to this project.

I agree to the point of removing unused framework ignores and coverage related things.

I think IDE ignores should be kept. If anything, missing IDE(s) ignores should be added. It would be nice to have them in gitignore so that any beginner level contributors(advanced coders accidentally) don't add unwanted files to a commit which needs to be pointed out in the PR later. It's not big deal, but helps avoid some back forth on simple PRs.

Reply:
Added IDEs, kept envs, other is left unchanged for this commit.